### DISCUSSION TITLE
Adding a Daily Sales Overview

### DISCUSSION THREAD

**Discussion by zGamerZz** [DoctorFuchs/kiosk#61] [Ideas]
I know that this project didn´t update since 2022...

but it would be nice if you can implement a feature that logs every sale in real-time, including details such as product name, quantity sold, total price, and timestamp. This information would be used to generate a daily sales overview displayed on a daily_sales.html page.

**Comment by DoctorFuchs** [comment]
Hello @zGamerZz,

I am very surprised you found this repository. As you already noticed, this project is inactive for some time. 
However there is a private follow up project. I will ask my team members, for permission to make it public/give you access.
But there is no such daily sales overview in neither of this projects, however the follow up project contains possiblities to create such a feature.

If you need access for a educational purposes, please tell me. 

I wish you a great christmas. 

Greetings
DoctorFuchs

Edit: Greetings

**Reply by zGamerZz** [reply]
Oh, okay, thank you for the information! I would appreciate it if I could get access to the repository. I need it for my school kiosk because we want to digitalize the system.

Thank you very much!

(By the way, I also speak German, in case that's easier for you.)

Best regards,
zGamerZz

**Reply by DoctorFuchs** [reply]
I am sorry for my late response. I hope you had a fantastic time with your family and friends. 
We really want to work with you on this feature. To make this possible, I would recommend to communicate further informations on Discord.

You can add me as friend (Username is the same on DC).

Thank you for being patient. I wish you a happy new year.

Doctorfuchs

(As you already noticed, I speak german. In this thread/repo I continue speaking english, just to give any other person the chance to understand what are we talking about)
